app/helpers/api/services_helper.rb

```diff
@@ -46,7 +46,7 @@ def friendly_service_setting service, setting
   end
 
   def delete_service_link(service, options = {})
-    msg = t('api.services.forms.definition_settings.delete_confirmation', name: h(service.name))
+    msg = t('api.services.forms.definition_settings.delete_confirmation', name: j(service.name))
     delete_link_for(admin_service_path(service), {data: { confirm: msg }, method: :delete}.merge(options) )
```

Comment:
Leave above as is and  escaping the whole message `msg` here?

Reply:
Right, better! :+1:

Reply:
Waaait, maybe there's no need :thinking: because the rest of the message is what we write ourselves in the locales, so we may wanna add some HTML there on purpose and it is a trustable source :thinking:

Reply:
Yes but we need to escape the name anyway

Reply:
@hallelujah , sorry but I am not understanding you right now :thinking:

This commit right now escapes the name only which is what is introduced from the user from outside, then you suggested to escape the whole message (`msg`), which contains the message written by ourselves in the locales + the name (in this commit, the name is escaped but not the rest of the message). So I said that maybe we shouldn't escape the whole message because the rest of the message is what we wrote ourselves in the locales so it is trustable and we also might include some HTML there intentionally.